tests/testsuite/build.rs

```diff
@@ -3976,11 +3976,13 @@ fn compiler_json_error_format() {
             name = "foo"
             version = "0.5.0"
             authors = ["wycats@example.com"]
+            build = "build.rs"
```

Comment:
This line is not neccessary, `build.rs` is the default name for quite some time now.